### ISSUE TITLE
♿ Make validator explicitly look for `lang` attribute on document and `alt` on images?

### ISSUE THREAD

**Issue by TetraLogicalHelpdesk** [ampproject/amphtml#32780]
## Describe the new feature or change to an existing feature you'd like to see

A large part of the relatively trivial accessibility issues found during the recent audits of AMP components and documentation was the general lack of `lang` attributes on the `<html>` element and the lack of `alt` attributes on image elements (`<amp-img>`. `amp-anim`, etc).

These are the sort of omissions that are flagged as warnings by the regular [W3C HTML validator](https://validator.w3.org/nu/).

Would it make sense to have the AMP validator to also look for these relatively straightforward things (noting that `alt` text - and `alt=""` empty attribute to explicitly denote purely presentational images - is in general required for [WCAG 2.1 SC 1.1.1](https://www.w3.org/TR/WCAG/#non-text-content), and use of `lang` for [WCAG 2.1 SC 3.1.1](https://www.w3.org/TR/WCAG/#language-of-page)

**Comment by kristoferbaxter** [comment]
Since this would be a breaking change for existing documents, is there another pathway to give document developers this information?

Could we implement Pixi rules (page experience) since these directly relate to the page experience many visitors have?

Adding @sebastianbenz and @patrickkettner for thoughts before potentially transferring the issue to `amp.dev`.

**Comment by sebastianbenz** [comment]
I like the idea of making this a Pixi check. However, this would mean that we're starting suggesting things which are not officially part of PX. This could give the wrong impression. What I could imagine though is having another category of recommendations which are somehow marked as "optional". @patrickkettner @nainar wdyt?

**Comment by caroqliu** [comment]
+1 This fits well with Pixi but could confuse the intent around PX. 

As an alternative, could these surface as a non-blocking AMP validator warnings?

**Comment by honeybadgerdontcare** [comment]
> As an alternative, could these surface as a non-blocking AMP validator warnings?

I imagine that would create a lot of warnings for a lot of sites and flood Search Console and make more target warnings lost in the sea.

**Comment by kristoferbaxter** [comment]
Agree with @honeybadgerdontcare, validator messages to search console are more specifically used for broken functionality or items making pages invalid.

Moving the messaging to the Page Experience tool seems like a good compromise.
